app/app.rb

```diff
@@ -0,0 +1,66 @@
+require 'sinatra/base'
+require "sinatra/json"
+require './lib/menu'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.